lib/utils/ruleRawsAfterIndex.mjs

```diff
@@ -1,7 +1,13 @@
 /**
- * @param {import('postcss').Rule} rule
+ * @param {import('postcss').Rule} node
  * @returns {number}
  */
-export default function ruleRawsAfterIndex(rule) {
-	return rule.source?.end?.offset ?? rule.toString().length;
+export default function ruleRawsAfterIndex(node) {
+	// always subtract 1 for `}`
+
+	if (!node.source?.end?.offset) return node.toString().length - 1;
+
+	if (!node.raws?.after?.length) return node.source.end.offset - 1;
+
+	return node.source.end.offset - (node.raws.after.length + 1);
```

Comment:
```suggestion
export default function ruleRawsAfterIndex(node) {
	const endOffset = node.source?.end?.offset;
	const afterLength = node.raws?.after?.length;
	const closingParenthesisLength = 1;

	if (!offset) return node.toString().length - closingParenthesisLength;

	if (!afterLength) return endOffset - closingParenthesisLength;

	return endOffset - afterLength + 1;
```

All in all, I am not sure which one is better.
You can disregard it if you don't like it.